L/libgmime/build_tarballs.jl

```diff
@@ -0,0 +1,48 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libgmime"
+version = v"3.2.15"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/jstedfast/gmime/releases/download/$(version)/gmime-$(version).tar.xz", "84cd2a481a27970ec39b5c95f72db026722904a2ccf3fdbd57b280cf2d02b5c4")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd gmime-3.2.15/
+echo '#define ICONV_ISO_INT_FORMAT "iso-%u-%u"' > iconv-detect.h
+echo '#define ICONV_ISO_STR_FORMAT "iso-%u-%s"' >> iconv-detect.h
+echo '#define ICONV_SHIFT_JIS "shift-jis"' >> iconv-detect.h
+echo '#define ICONV_10646 "UCS-4BE"' >> iconv-detect.h
+
+export LDFLAGS="$LDFLAGS -liconv"
+CMAKE_EXE_LINKER_FLAGS="-Wl,--copy-dt-needed-entries"
+export CMAKE_EXE_LINKER_FLAGS
```

Comment:
Can you please elaborate why this is needed?
```suggestion
export CMAKE_EXE_LINKER_FLAGS="-Wl,--copy-dt-needed-entries"
```

Reply:
Please answer the question.

Reply:
This fixes issue with [error adding symbols: DSO missing from command line](https://stackoverflow.com/questions/19901934/libpthread-so-0-error-adding-symbols-dso-missing-from-command-line)

Reply:
I still don't get this one.  Can you please provide the full compiler invocation and related error message?

Reply:
Hello,
Here is error message:
```
/opt/x86_64-linux-gnu/bin/../lib/gcc/x86_64-linux-gnu/8.1.0/../../../../x86_64-linux-gnu/bin/ld: charset-map.o: undefined reference to symbol 'libiconv'
/opt/x86_64-linux-gnu/bin/../lib/gcc/x86_64-linux-gnu/8.1.0/../../../../x86_64-linux-gnu/bin/ld: /workspace/destdir/lib/libiconv.so.2: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
make[3]: *** [Makefile:817: charset-map] Error 1
make[3]: *** Waiting for unfinished jobs....
```

Documentstion also [says](https://fossies.org/linux/gmime/docs/reference/html/gmime-building.html#:~:text=The%20GNU%20libiconv%20library%20is%20needed%20to%20build%20GLib%20and%20GMime%20if%20your%20system%20doesn%27t%20already%20have%20the%20iconv()%20function%20for%20doing%20conversion%20between%20character%20encodings.%20Most%20modern%20systems%20should%20have%20iconv().) that `libiconv` needed if the system doesn't have `iconv`